examples/shell/shell_common/cmd_otcli.cpp

```diff
@@ -185,12 +190,14 @@ void cmd_otcli_init()
 {
 #if CHIP_ENABLE_OPENTHREAD
 #if CHIP_TARGET_STYLE_EMBEDDED
+#if defined(__ZEPHYR__)
 #if OPENTHREAD_API_VERSION >= 85
     otCliInit(otInstanceInitSingle(), &OnOtCliOutput, NULL);
```

Comment:
What's the material difference between `otCliInit` and `otAppCliInit`? I couldn't find any documentation anywhere...